daemon/api_general.go

```diff
@@ -53,9 +54,11 @@ var (
 	}
 
 	sysInfoCmd = &Command{
-		Path:       "/v2/system-info",
-		GET:        sysInfo,
-		ReadAccess: interfaceOpenAccess{Interfaces: []string{"snap-interfaces-requests-control"}},
+		Path:        "/v2/system-info",
+		GET:         sysInfo,
+		POST:        sysInfoPost,
+		ReadAccess:  interfaceOpenAccess{Interfaces: []string{"snap-interfaces-requests-control"}},
+		WriteAccess: interfaceOpenAccess{Interfaces: []string{"snap-interfaces-requests-control"}},
```

Comment:
Does it make sense to constrain write access or to even call this from snapd-snap socket?

Reply:
no, this should be `openAccess{}` (i.e. open to anyone but excludes access over snapd-snap socket)

Reply:
this is done now